cloudify/workflows/workflow_context.py

```diff
@@ -306,7 +306,7 @@ def node_id(self):
     @property
     def relationships(self):
         """The node relationships"""
-        return self._relationship_instances.itervalues()
+        return iter(self._relationship_instances.values())
```

Comment:
those need to stay iterators because unfortunately things use this as iterators by calling `next`.
It would be better if usage was to just `for x in ..` instead of an explicit `next()`, but for now, let's keep back-compat